project/sites/pacni/themes/pacni_theme/src/scss/components.scss

```diff
@@ -5,6 +5,18 @@
 @import 'init/index';
 @import '../../node_modules/nicsdru_unity_theme/src/scss/3_components/index';
 
+#header {
+  background-image: url("http://pacni.gov.uk.lndo.site/sites/pacni/themes/pacni_theme/images/header_bg.jpg");
```

Comment:
You should be able to do this:
#header {
    background: resolve('header_bg.png') no-repeat top right;
}

I think the resolve will look in your local images folder.

Reply:
Thanks that worked great. Code now updated